test/mocha/release.js

```diff
@@ -8,7 +8,7 @@ function run(command, args, done) {
         process.stdout.write("\0");
     }, 5 * 60 * 1000);
     spawn(command, args, {
-        stdio: "ignore"
+        stdio: [ "ignore", 1, 2 ]
```

Comment:
Here I'm ignoring `stdin`, and pipe both `stdout` and `stderr` out.

Used to be I just ignore them all to avoid getting flooded on the console, but now that I'm comfortable enough navigating around them, I think it makes sense to have Travis CI record these details down, so we can have extra data points for diagnosis and even say performance tracking via `test/benchmark.js` timings.